sdk/storage/Azure.Storage.Blobs/src/BlobBaseClient.cs

```diff
@@ -1844,11 +1847,11 @@ static Task WritePartitionAsync(Response<BlobDownloadInfo> response, Stream dest
         /// a failure occurs.
         /// </remarks>
         public virtual Response Delete(
-            DeleteSnapshotsOption? deleteOptions = default,
+            bool? includeSnapshots = true,
```

Comment:
Is there any reason these have to be nullable?  I'd rather have it just `bool includeSnapshots = true` if that's a possibility.